commit ca26590ccc73a0c9ca625acab5f2acaa08408d7d
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Wed May 27 19:17:03 2020 -0700

    Fix linking issue
    
    /usr/bin/ld: /mnt/cold/sources/vboot_reference/build/libvboot_util.a(subprocess.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:93: multiple definition of `subprocess_stdin'; /mnt/cold/sources/vboot_reference/build/libvboot_util.a(crossystem.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:93: first defined here
    /usr/bin/ld: /mnt/cold/sources/vboot_reference/build/libvboot_util.a(subprocess.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:103: multiple definition of `subprocess_stderr'; /mnt/cold/sources/vboot_reference/build/libvboot_util.a(crossystem.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:103: first defined here
    /usr/bin/ld: /mnt/cold/sources/vboot_reference/build/libvboot_util.a(subprocess.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:88: multiple definition of `subprocess_null'; /mnt/cold/sources/vboot_reference/build/libvboot_util.a(crossystem.o):/home/anatol/sources/vboot_reference/host/lib/include/subprocess.h:88: first defined here
    
    Signed-off-by: Anatol Pomozov <anatol.pomozov@gmail.com>

diff --git a/host/lib/include/subprocess.h b/host/lib/include/subprocess.h
index eaf75502..b24b987f 100644
--- a/host/lib/include/subprocess.h
+++ b/host/lib/include/subprocess.h
@@ -85,22 +85,22 @@ struct subprocess_target {
 /**
  * A convenience subprocess target which uses TARGET_NULL.
  */
-struct subprocess_target subprocess_null;
+extern struct subprocess_target subprocess_null;
 
 /**
  * A convenience subprocess target which uses TARGET_FD to STDIN_FILENO.
  */
-struct subprocess_target subprocess_stdin;
+extern struct subprocess_target subprocess_stdin;
 
 /**
  * A convenience subprocess target which uses TARGET_FD to STDOUT_FILENO.
  */
-struct subprocess_target subprocess_stdout;
+extern struct subprocess_target subprocess_stdout;
 
 /**
  * A convenience subprocess target which uses TARGET_FD to STDERR_FILENO.
  */
-struct subprocess_target subprocess_stderr;
+extern struct subprocess_target subprocess_stderr;
 
 /**
  * Call a process and run until completion.
